README.rst

```diff
@@ -27,15 +27,29 @@ And an Array can be converted to a numpy ndarray on demand::
     array([ 0.72151309,  0.54654914], dtype=float32)
 
 
-Get in touch!
--------------
+------------------------------------------
+Important Note : Future development status
+------------------------------------------
+Biggus was originally developed primarily to provide large data handling for
+`Iris <http://github.com/SciTools/iris>`_ .
+
+For most applications it has now been surpassed by
+`Dask <http://https://github.com/dask/dask>`_, which is used by the majority
+of other projects working with virtual arrays.
+
+Thus, **Dask is now generally preferred to Biggus**.
+
+Iris itself now uses Dask in place of Biggus, from version 2.0 onwards.
+As Biggus is little used by projects other than Iris, this means that its
```

Comment:
You could drop "this means that" from this sentence and I reckon it would make the same sense and possibly be easier to read.